www/src/pages/en/usage/prisma.md

```diff
@@ -16,11 +16,11 @@ You will find the Prisma schema file at `/prisma/schema.prisma`. This file is wh
 
 ### With NextAuth.js
 
-When you select NextAuth.js in combination with Prisma, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account` and `VerificationToken` models, as per the [NextAuth.js documentation](https://next-auth.js.org/adapters/prisma).
+When you select NextAuth.js in combination with Prisma, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account`, and `VerificationToken` models, as per the [NextAuth.js documentation](https://next-auth.js.org/adapters/prisma).
 
 ## Default Database
 
-The default database is a SQLite database, which is great for development and quickly spinning up a proof-of-concept, but not recommended for production. You can change the database to use by changing the `provider` in the `datasource` block to either `postgresql` or `mysql`, and then updating the connection string within environment variables to point to your database.
+The default database is an SQLite database, which is great for development and quickly spinning up a proof-of-concept but is not recommended for production. You can change the database to use by changing the `provider` in the `datasource` block to either `postgresql` or `mysql`, and then updating the connection string within environment variables to point to your database.
```

Comment:
a SQLite database is correct no?

Reply:
![image](https://user-images.githubusercontent.com/89210438/202669521-706c7b06-bab1-4bf4-b5cb-cfcf7ee0c9e2.png)

Refer: https://www.grammar.com/a-vs-an-when-to-use

Reply:
It you don't say S Q Lite do you? Or is that just me

Reply:
Is there a "correct" pronunciation? Been saying S Q Lite all this time

Reply:
![image](https://user-images.githubusercontent.com/89210438/202674427-5a4bf5bb-b22f-49f5-a72c-19ae4b93f245.png)

Refer: https://database.guide/is-it-pronounced-s-q-l-or-sequel/